src/main/resources/gov/nasa/jpl/mbee/mdk/options/EnvironmentOptionsResources.properties

```diff
@@ -10,6 +10,8 @@ CUSTOM_USER_SCRIPT_DIRECTORIES_ID=Custom User Scripts Directories
 CUSTOM_USER_SCRIPT_DIRECTORIES_ID_DESCRIPTION=Specifies additional directories that custom user scripts can be found in. Multiple directories can be specified, but they must be separated by ";" on Windows systems or ":" on Mac and Linux systems.
 SHOW_ADVANCED_OPTIONS_ID=Show Advanced Options
 SHOW_ADVANCED_OPTIONS_ID_DESCRIPTION=Enables advanced features in the MMS menu, such as branch validation. You must restart MagicDraw after enabling this field to see the options in the menu.
+DOCBOOK_TO_PDF_STYLESHEET=DocBook to PDF Stylesheet
+DOCBOOK_TO_PDF_STYLESHEET_DESCRIPTION=DocBook to PDF Stylesheet
```

Comment:
Replace placeholder with a description of the option.

Reply:
Changed DOCBOOK_TO_PDF_STYLESHEET_DESCRIPTION=Stores the last DocBook to PDF Stylesheet used while converting DocBook to PDF.   Is this acceptable?